build-support/bin/release.sh

```diff
@@ -561,32 +561,19 @@ function build_pex() {
   dest_suffix="py36.pex"
   case "${mode}" in
     build)
-      case "$(uname)" in
-        # NB: When building locally, we use a platform that does not refer to the ABI version, to
-        # avoid needing to introspect the python ABI for this machine.
-        Darwin)
-          local platform="${osx_platform_noabi}"
-          ;;
-        Linux)
-          local platform="${linux_platform_noabi}"
-          ;;
-        *)
-          echo >&2 "Unknown uname"
-          exit 1
-          ;;
-      esac
-      local platforms=("${platform}")
+      # NB: When building locally, we explicitly target our local Py3.
+      local platform_flags=("--python=$(which "$PY")")
```

Comment:
Just `$PY` should be enough, but if we already have a dependency on `which` this is fine.

Reply:
Consider naming these `target_flags` or `distribution_target_flags` or `dustribution_targets`. Getting out of Pants mode, target (DistributionTarget) is the name Pex unifies local interpreter vs foreign platform under.

Reply:
I think Shellcheck will make you use the more portable `command -v` over `which`.

Reply:
That's what I get for skipping the hooks.